appveyor.yml

```diff
@@ -17,7 +17,7 @@ cache:
 
 environment:
   STACK_ROOT: C:\sr
-  STACK_VERSION: 2.1.1
+  STACK_VERSION: 2.5.1
```

Comment:
I see that Stack 2.5.1 is on Chocolatey, so this should work 👍 
Just a lot of places needed to be look before bumping versions of tools everywhere...

* https://chocolatey.org/packages/haskell-stack